src/main/kotlin/org/taktik/icure/be/ehealth/logic/kmehr/sumehr/impl/v20161201/SumehrExport.kt

```diff
@@ -468,8 +506,39 @@ class SumehrExport : KmehrExport() {
 			} else {
 				getAssessment(trn).headingsAndItemsAndTexts
 			}
-			createItemWithContent(eds, items.size + 1, "healthcareelement", listOf(makeContent("fr", Content(eds.descr))).filterNotNull())?.let {
-				items.add(it)
+
+			// familyrisk not allowed in Sumehr
+			val familyrisk = eds.tags?.find {it.type == "CD-ITEM" && it.code == "familyrisk"}
+			if(familyrisk != null){
+				familyrisk.code = "healthcareelement"
+			}
+
+			listOf("healthcareelement", "allergy", "adr", "risk", "socialrisk").forEach { edType ->
+				if(eds.tags?.find {it.type == "CD-ITEM" && it.code == edType} != null){
+					if(edType == "healthcareelement"){
+						//EhValidator-v3.0.0: For "healthcareelement", "Codification / Version" used in "content" elements must be one of these: "ICPC 2", "ICD 10"
+						eds.codes = eds.codes.filter { c -> c.type == "ICPC" || c.type == "ICD" }.toSet()
```

Comment:
BE-THESAURUS codes (which are in fact IBUI codes) can be mapped to CD-CLINICAL content elements in KMEHR.

https://www.ehealth.fgov.be/standards/kmehr/en/tables/ibui